pkgs/by-name/ta/taxi/package.nix

```diff
@@ -1,58 +1,56 @@
 {
   lib,
   stdenv,
+  desktop-file-utils,
   fetchFromGitHub,
   gobject-introspection,
-  gtk3,
+  gtk4,
+  libadwaita,
   libgee,
   libhandy,
   libsecret,
-  libsoup_2_4,
+  libsoup_3,
   meson,
   ninja,
   nix-update-script,
   pantheon,
   pkg-config,
   python3,
   vala,
-  wrapGAppsHook3,
+  wrapGAppsHook4,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "taxi";
-  version = "2.0.2";
+  version = "2.0.2-unstable-2024-12-26";
 
   src = fetchFromGitHub {
     owner = "Alecaddd";
     repo = "taxi";
-    rev = version;
-    sha256 = "1a4a14b2d5vqbk56drzbbldp0nngfqhwycpyv8d3svi2nchkvpqa";
+    rev = "b1c81490641f102005d9451a33d21610c0637e22";
+    sha256 = "sha256-boPwRSHzFpbrzRoSmNWf/fgi3cJDEt9qjZHWQWutL+o=";
   };
 
   nativeBuildInputs = [
+    desktop-file-utils
     gobject-introspection
     meson
     ninja
     pkg-config
     python3
     vala
-    wrapGAppsHook3
+    wrapGAppsHook4
   ];
 
   buildInputs = [
-    gtk3
+    gtk4
+    libadwaita
     libgee
     libhandy
```

Comment:
I guess this is no longer needed with the gtk4 port.

Reply:
(Note that this is a library for gtk3 apps so it is mostly 100% a bug if it is referenced anywhere in a gtk4 app)

Reply:
Seems `libgee` isn't needed either, so can just remove all of that stuff. I put gtk4 back.